client/src/components/Datatypes/model.ts

```diff
@@ -39,4 +39,14 @@ export class DatatypesMapperModel {
     isSubTypeOfAny(child: string, parents: DatatypesCombinedMap["datatypes"]): boolean {
         return parents.some((parent) => this.isSubType(child, parent));
     }
+
+    /** For classes like `galaxy.datatypes.{parent}.{extension}`, get the extension's parent */
+    getParentDatatype(extension: string) {
```

Comment:
That logic is redundant and also doesn't reflect how datatypes match. Is there a problem with using `isSubTypeOfAny` ?